### ISSUE TITLE
Too many UIA events cause NVDA to repeatedly report object information

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#14145]
Use Office Tool Plus to reproduce the problem, the following is the download link:
https://github.com/YerongAI/Office-Tool/releases/download/v9.0.3.7/Office_Tool_with_runtime_v9.0.3.7.7z
### Steps to reproduce:
1. Download and unzip the attachment;
2. Run the Office Tool Plus executable;
3. Use the Tab key to navigate to the 'home' radio button;
4. Press the up and down arrows to move through the set of radio buttons;
5. In the nvda advanced settings panel: Registration for UI Automation events and property changes:: "Global" or "Automatic (prefer selective)"

### Actual behavior:
NVDA reports too much information:
Home radio button checked 转到页面
blank

Home 
Home 

Deploy radio button not checked 转到页面

blank

Deploy 
Deploy 
Activate radio button not checked 转到页面
blank
Activate 
Activate 
### Expected behavior:
NVDA should be able to report the above information more concisely, at least it should not be repeated.

### NVDA logs, crash dumps and other attachments:
The following logs were recorded with the EventTracker add-on enabled.
[log.txt](https://github.com/nvaccess/nvda/files/9564572/log.txt)

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Version: alpha-26566,aaa0fd11 (2022.4.0.26566)

#### Windows version:
Windows 10 21H2 (x64) build 19044.1889
#### Name and version of other software in use when reproducing the issue:
Office Tool Plus9.0.3.7
#### Other information about your system:
None
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
NVDA2022.2.2 cannot reproduce this
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by cary-rowen** [comment]
This is very similar to what I comment in #14086, they may be related.

**Comment by mazen428** [comment]
Most of them are item status events, which should have been fixed by the revert
https://github.com/nvaccess/nvda/pull/14136

**Comment by cary-rowen** [comment]
I can also reproduce this with the latest alpha-26572, #14136 has been merged. This may not solve the problem.

**Comment by josephsl** [comment]
Hi, do you happen to have Windows App Essentials add-on installed as well? See if disabling that add-on fixes it, and if yes, looks like I need to remove item status handler from that add-on altogether (the proposed solution of limiting item status changes to focused control may not be effective after all). I’m also wondering if the app in question is also a WPF app, and if yes, looks like it might be time to have another discussion about UIA events with Microsoft folks. Thanks.

**Comment by josephsl** [comment]
Hi,

Also, see if Windows App Essentials September 14th build helps as it will limit item status event to focused control. If this does not work (obviously after enabling the add-on and restarting NVDA), then I'll remove UIA item status event handler from my add-on this weekend.

Thanks.

**Comment by cary-rowen** [comment]
@josephsl 
Hello, the problem is indeed caused by the wintenApps add-on.
But I installed the 9.14-Dev version of the add-on and the problem was solved.

**Comment by josephsl** [comment]
Hi,

Try build: https://github.com/josephsl/wintenApps/releases/download/22.09/wintenApps-try-114145-20220914.nvda-addon

The difference between September 14th dev build and the linked try build is UIA item status event handler - just system focus will be checked instead of checking both focus and pending events. This is a slightly modified version of the PR that was reverted yesterday - the reverted pull request did not check system focus, leading to handling item status event from everywhere.

Let me know if the try build also fixes the problem reported in this issue, and if yes, I'll create a follow-up pull request restoring UIA item status event handler with focus check included.

Thanks.

**Comment by cary-rowen** [comment]
@josephsl 
Hello,
I tested that the try-build version you provided also solves the problem.
Thanks

**Comment by josephsl** [comment]
Hi, okay, I’ll include this change as part of next Windows App Essentials development build (this weekend), and things work well for people, I’ll come up with a PR to reintroduce item status event handler with a change to limit this to focused control (unless the problem lies deeper within Visual Studio itself). Thanks.

**Comment by cary-rowen** [comment]
As far as my original comment is concerned, it's resolved for now and I'm closing this.
